bluebottle/categories/models.py

```diff
@@ -4,17 +4,27 @@
 from parler.models import TranslatableModel, TranslatedFields
 
 from bluebottle.clients import properties
+from bluebottle.files.validators import validate_video_file_size
 from bluebottle.utils.fields import ImageField
 from adminsortable.admin import SortableMixin
 
 
 class Category(TranslatableModel):
     slug = models.SlugField(_('slug'), max_length=100, unique=True)
 
-    image = ImageField(_("image"), max_length=255, blank=True, null=True, upload_to='categories/',
-                       help_text=_("Category image"))
-    image_logo = ImageField(_("logo"), max_length=255, blank=True, null=True, upload_to='categories/logos/',
-                            help_text=_("Category Logo image"))
+    image = ImageField(
+        _("image"), max_length=255, blank=True, null=True,
+        upload_to='categories/',
+        help_text=_("Category image"))
+    video = models.FileField(
+        _("video"), max_length=255,
+        blank=True, null=True,
+        validators=[validate_video_file_size],
```

Comment:
This will need an extra validator that validates the mimetype of this field. If not, this results in a security issue, where malicious files can be uploaded.